src/Widgets/ColumnContainerWidget/ColumnContainerWidgetComponent.tsx

```diff
@@ -5,29 +5,43 @@ provideComponent(ColumnContainerWidget, ({ widget }) => {
   const columns = widget.get('columns')
   if (!columns.length) return null
 
-  const content = columns.map((columnWidget, index) => {
-    const colSize = columnWidget.get('colSize') || 1
-    const className = widget.get('disableResponsiveAdaption')
-      ? `col-${colSize}`
-      : `col-md-${colSize}`
-    return (
-      <div key={index} className={className}>
-        <ContentTag
-          content={columnWidget}
-          attribute="content"
-          className="h-100"
-        />
-      </div>
-    )
-  })
+  const alignment = widget.get('alignment') || 'start'
+  const isFlex = widget.get('layoutMode') === 'flex'
 
-  const classNames = ['row']
+  const classNames = [`align-items-${alignment}`, isFlex ? 'd-flex' : 'row']
 
-  if (widget.get('alignment')) {
-    classNames.push(`align-items-${widget.get('alignment')}`)
-  } else {
-    classNames.push('align-items-start')
-  }
+  return (
+    <div className={classNames.join(' ')}>
+      {columns.map((columnWidget) => {
+        const key = columnWidget.id()
 
-  return <div className={classNames.join(' ')}>{content}</div>
+        if (isFlex) {
+          const flexGrow = columnWidget.get('flexGrow')
+          const className = flexGrow ? 'flex-grow-1' : undefined
+          return (
+            <ContentTag
+              key={key}
+              className={className}
+              content={columnWidget}
+              attribute="content"
+            />
+          )
+        }
```

Comment:
I think it would be useful, if there is some spacing between the columns. The search input and the button of the next screenshot are placed in a flex column widget:

![Screenshot 2024-07-31 at 11 41 04](https://github.com/user-attachments/assets/d81b9113-34a3-4697-82b1-7c69151f9ba9)